ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -88,25 +90,20 @@ Logout From RHODS Dashboard
     [Documentation]  Logs out from the current user in the RHODS dashboard
     ...    This will reload the page and show the `Log in with OpenShift` page
     ...    so you want to use `Login to RHODS Dashboard` after this
-    # Another option for the logout button
-    #${user} =  Get Text  xpath:/html/body/div/div/header/div[2]/div/div[3]/div/button/span[1]
-    #Click Element  xpath://span[.="${user}"]/..
-    ${version_check}=  Is RHODS Version Greater Or Equal Than  1.21.0
-    IF  ${version_check}==True
-        Click Button  xpath://button[@id="user-menu-toggle"]
-    ELSE
-        Click Button  xpath:(//button[@id="toggle-id"])[2]
-    END
-    Wait Until Page Contains Element  xpath://a[.="Log out"]
-    Click Element  xpath://a[.="Log out"]
+    Click Button  ${USER_MENU_TOGGLE}
+    Wait Until Page Contains Element  xpath:${LOGOUT_BTN}
+    Click Element  xpath:${LOGOUT_BTN}
```

Comment:
Does this depend on the version of openshift in use?
![image](https://github.com/user-attachments/assets/4432d913-901c-4e42-96a4-6fd973a9de1c)

Reply:
never mind again, ofc it does not